user/howto/launch-dask-gateway-cluster.md

```diff
@@ -0,0 +1,225 @@
+# Launch a dask-gateway cluster
+
+This guide shows you how to launch a Dask gateway cluster for parallel and distributed computing.
+
+:::{contents}
+:depth: 2
+:local:
+:::
+
+## What is Dask Gateway?
+
+Dask Gateway allows users to launch clusters for scaling computations efficiently with more CPU and memory on cloud resources, without requiring direct access to the underlying Kubernetes backend of the 2i2c hub. Configuration, such as efficient cluster resourcing, authentication and security settings, is automatically handled for users to provide a consistent user experience across the hub.
+
+:::{admonition} What's the difference between Dask and Dask Gateway
+:class: note
+When using Dask on a local machine, you will often instantiate a cluster using
+```python
+from dask.distributed import LocalCluster
+cluster = LocalCluster()
+client = cluster.get_client()
+```
+For a 2i2c Dask hub, we instantiate a cluster with Dask Gateway that requests cloud resources with Kubernetes from a shared, centrally managed cluster environment.
+:::
+
+## Usage
+
+This section roughly follows the [Dask Gateway Docs – Usage](https://gateway.dask.org/usage.html) together with the [Training on Large Datasets](https://examples.dask.org/machine-learning/training-on-large-datasets.html) example.
+
+:::{warning}
+Avoid large, long-running, idle clusters, which consume cloud computing resources and costs. Only use a cluster when you need it and shutdown when not in use.
+:::
+
+### Connect to a dask-gateway server
+
+Create a gateway client to communicate with the dask-gateway server.
+
+```python
+from dask_gateway import Gateway
+gateway = Gateway()  # Use values stored in your local configuration (recommended)
```

Comment:
"stored in your local configuration"

This sounded a bit like it mean the community's pre-configured configuration or similarly - but this is clarified below in the note so its quite fine anyhow.

Reply:
```suggestion
gateway = Gateway()  # Uses values configured for the 2i2c Dask hub (recommended)
```

Ah, I think that was a copy-paste from the Dask Docs!